### PR TITLE
Support `zeebe:properties`

### LINKED ISSUE #30: Support `zeebe:property`
__Is your feature request related to a problem? Please describe__

Support for the new [`zeebe:property`](https://github.com/camunda/zeebe/issues/9868) property.

__Describe the solution you'd like__

* [ ] `zeebe:property` is recognized by moddle

__Describe alternatives you've considered__

None.

----

Related to https://github.com/camunda/zeebe/issues/9868
Related to https://github.com/camunda/product-hub/issues/335

### PR DESCRIPTION
**Pull request by nikku**
Provides support for the newly added [`zeebe:properties` extension point](https://github.com/camunda/zeebe/issues/9868).

----

Closes #30

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,3842 @@
 {
   "name": "zeebe-bpmn-moddle",
   "version": "0.12.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (pinussilvestrus) [inline package-lock.json:4]: Are we using lock file version 2 now?

Comment (barmac) [reply package-lock.json:4]: I believe this is an oversight. The npm@8 PRs are still drafts.

File: test/spec/xml/read.js [hunk 2]
```diff
@@ -81,29 +81,39 @@ describe('read', function() {
     });
 
 
-    describe('zeebe:Subscription', function() {
```
Comment (barmac) [inline test/spec/xml/read.js:84]: I don't quite get it. Is the subscription gone? Or is this an oversight that the test case is removed?

Comment (pinussilvestrus) [reply test/spec/xml/read.js:84]: It was a duplicate: https://github.com/camunda-cloud/zeebe-bpmn-moddle/pull/31/commits/441d76ad60c7a43a618ea1edeadeefe690c759f7

And GitHub uses it as replacement 😄

Comment (barmac) [reply test/spec/xml/read.js:84]: 🤦 on me

**Review by barmac — APPROVED** [review]

**Comment by barmac** [comment]
I removed the package-lock update so that we can release without forcing npm@8 yet.

**Comment by barmac** [comment]
Released as v0.13.0

File: package-lock.json (hunk 1 above)
Comment (nikku) [reply package-lock.json]: I was already preparing for tomorrows decision. Actually intentional. :wink:

File: test/spec/xml/read.js (hunk 2 above)
Comment (nikku) [reply test/spec/xml/read.js:84]: :+1:

**Pull request merged**